src/teuthology_api/services/kill.py

```diff
@@ -26,38 +28,48 @@ def run(args, send_logs: bool, access_token: str, request: Request):
     run_name = args.get("--run")
     if run_name:
         run_details = get_run_details(run_name)
-        run_owner = run_details.get("user")
+        jobs_details = run_details.get("jobs", [])
```

Comment:
I saw this line and compared it with the line above `if response_org_dic.get("state") == "active":`

We could have some consistency? unless there is a good reason why some `get` you include a default argument